doc/developing-packages.rst

```diff
@@ -259,6 +259,54 @@ your libraries and executable sections. For example:
 Note that the ``import`` **must** be the first thing in the stanza. For more
 information see the :ref:`common-stanzas` section.
 
+Sets of packages that are verified to build together
+----------------------------------------------------
+
+Since Cabal 3.8, you can import from a
+:ref:`remote project configuration file<conditionals and imports>`:.
+This frees you from having to determine the acceptable version bounds for your
+dependencies, as described in the previous section.
+
```

Comment:
This doesn't free you from anything. It just lets you further constrain things.

Reply:
This was removed in the revised PR.